src/legacy/ui/public/vis/editors/default/_sidebar.scss

```
@@ -221,3 +221,6 @@
   }
 }
 
+vis-agg-param-react-wrapper .euiFormRow {
+  margin-bottom: 7px;
```

Some SASS guidelines were missed here:
- Don't apply styles directly to EUI classes (just pass down a custom class name)
- Don't use pixel values (use an EUI SASS variable)
https://elastic.github.io/eui/#/guidelines/sass

```suggestion
  margin-bottom: $euiSizeS;
```